packages/flutter_tools/lib/src/resident_web_runner.dart

```diff
@@ -180,7 +180,7 @@ class ResidentWebRunner extends ResidentRunner {
           printStatus(message);
         }
       });
-      websocketUri = Uri.parse(_debugConnection.wsUri);
+      websocketUri = Uri.parse(_debugConnection.uri);
```

Comment:
Is this still a websocket uri, or did it change to http too?

Reply:
@jakemac53 Is this change correct? `wsUri` is gone from the API.

Reply:
if you can still run in intellij/vscode you'll know it works

Reply:
This will now sometimes be an SSE connection instead of a web socket (which I think we signify with a made up `sse:` scheme). You shouldn't have to worry about it right away but basically it is required to work over uberproxy.

Reply:
I'm not sure if IDEs will understand the sse connection. Is there anyway to induce this to check?

Reply:
You can check if the scheme of the uri is `ws` or `wss`, that means its a web socket. For now I think it would be fine to just assert that is the case?

Devtools has support for SSE now but you are correct nothing else will understand it.

Reply:
I don't expect this particular code to run in google3 for a while so it is probably fine to assert and move on then.